src/GitHub.Api/IO/NiceIO.cs

```diff
@@ -283,6 +283,8 @@ public bool DirectoryExists(string append = "")
 
         public bool DirectoryExists(NPath append)
         {
+            if (append == null)
```

Comment:
`Combine` doesn't care if it's null so it wouldn't blow up, but it's a waste of resources creating new instances of the same thing over and over again, `Combine` always creates a new thing.